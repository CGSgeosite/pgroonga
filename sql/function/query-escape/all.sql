SELECT pgroonga.query_escape('a+B-c< >あいう~*()"\\'':');
