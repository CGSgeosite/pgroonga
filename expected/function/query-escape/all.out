SELECT pgroonga.query_escape('a+B-c< >あいう~*()"\\'':');
            query_escape             
-------------------------------------
 a\+B\-c\< \>あいう\~\*\(\)\"\\\\'\:
(1 row)

